examples/solo8_vanilla/interactive_pos_control.py

```diff
@@ -0,0 +1,49 @@
+"""A demo for the Solo8 v2 Vanilla with configurable position control on the
+the joints. 
+"""
+
+import gym
+import numpy as np
+
+from gym_solo.envs import solo8v2vanilla
+from gym_solo.core import obs
+from gym_solo.core import rewards
+from gym_solo.core import termination as terms
+
+
+if __name__ == '__main__':
+  config = solo8v2vanilla.Solo8VanillaConfig()
+  env: solo8v2vanilla.Solo8VanillaEnv = gym.make('solo8vanilla-v0', use_gui=True, realtime=True, config=config)
+
+  env.obs_factory.register_observation(obs.TorsoIMU(env.robot))
+  env.reward_factory.register_reward(1,rewards.UprightReward(env.robot))
+  env.termination_factory.register_termination(
+    terms.TimeBasedTermination(10000000))
```

Comment:
yeah just made a magic number so that the demo wouldn't time out. Would you prefer that I added a comment or just made a `Termination` which never ends (which is probably the correct way to do it)?

Reply:
We want the episode to end at some point right?

Reply:
Mentioned in the other PRs, but this is taken care of in #37

Reply:
Got it